# 射击机构数据具体数据

# 每秒冷却值
int32 coolingRate
# 热量上限
int32 coolingLimit
# 上限速度,单位m/s
int32 speedLimit